packages/link/package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "@mineral-ui/link",
+  "version": "0.0.0",
+  "description": "Link component",
+  "license": "Apache-2.0",
+  "main": "dist/umd/index.js",
+  "module": "dist/es/index.js",
+  "scripts": {
+    "build": "$npm_package_config_script_path/build.sh",
+    "build:analyze": "$npm_package_config_script_path/build-analyze.sh",
+    "clean": "$npm_package_config_script_path/clean.sh",
+    "commit": "$npm_package_config_script_path/commit.sh",
+    "eslint": "$npm_package_config_script_path/eslint.sh",
+    "flow": "$npm_package_config_script_path/flow.sh",
+    "jest": "$npm_package_config_script_path/jest.sh",
+    "lint": "$npm_package_config_script_path/lint.sh",
+    "pack-ls": "$npm_package_config_script_path/pack-ls.sh",
+    "start": "DEMO_LIST_PATH=\"link\" $npm_package_config_script_path/start.sh",
+    "test": "$npm_package_config_script_path/test.sh"
+  },
+  "devDependencies": {
+    "@mineral-ui/component-utils": "^0.3.0"
+  },
+  "peerDependencies": {
+    "@mineral-ui/component-utils": "^0.3.0",
+    "glamor": "^2.0.0",
+    "glamorous": "^3.0.0",
+    "react": "^15.0.0",
+    "react-dom": "^15.0.0",
+    "react-router-dom": "^4.1.2"
```

Comment:
er sorry, that makes it a devDependency. it does not need to be installed by a consumer